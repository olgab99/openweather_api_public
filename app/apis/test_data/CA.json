[
  {
    "city": "Morden",
    "id": 6078447
  },
  {
    "city": "Laval",
    "id": 6050612
  },
  {
    "city": "Thornhill",
    "id": 6165683
  },
  {
    "city": "Port Hope",
    "id": 6111881
  },
  {
    "city": "Wasaga Beach",
    "id": 6176421
  }
]