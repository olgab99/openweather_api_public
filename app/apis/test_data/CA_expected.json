[
   {
    "city": "Laval",
    "id": 6050612
  },
  {
    "city": "Morden",
    "id": 6078447
  },
   {
    "city": "Port Hope",
    "id": 6111881
  },
  {
    "city": "Thornhill",
    "id": 6165683
  },
  {
    "city": "Wasaga Beach",
    "id": 6176421
  }
]